proto/process/agent.proto

```diff
@@ -465,3 +465,35 @@ message Host {
 message DNSEntry {
   repeated string names = 1;
 }
+
+// reference https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto
+message Pod {
+	string name = 1;
+	string namespace = 2;
+	int64 age = 3;
```

Comment:
Any reason we can't just use a `createTime` here and calculate age ourselves?

Reply:
I'm going to rename this field it's going to be the creation timestamp indeed